dnf/repo.py

```diff
@@ -613,7 +614,7 @@ def _handle_load(self, handle):
 
     def _handle_load_core(self, handle):
         if handle.progresscb:
-            self._md_pload.start(self.name)
+            self._md_pload.start(self.name or self.id or 'unknown')
```

Comment:
Is it ever possible for `self.name` and `self.id` to both be `None`? If not, I don't think it makes sense to have the fallback `unknown` value.